client/internal/url/url.go

```diff
@@ -76,3 +77,23 @@ func ReadSmartLicense(baseUrl string) string {
 func ReadAccessPolicies(baseUrl string, domainUid string, limit int) string {
 	return fmt.Sprintf("%s/fmc/api/fmc_config/v1/domain/%s/policy/accesspolicies?limit=%d", baseUrl, domainUid, limit)
 }
+
+func CreateUser(baseUrl string, username string) string {
+	return fmt.Sprintf("%s/anubis/rest/v1/users/%s", baseUrl, username)
+}
+
+func ReadUserByUsername(baseUrl string, username string) string {
+	return fmt.Sprintf("%s/anubis/rest/v1/users?q=name:%s", baseUrl, username)
+}
+
+func UserByUid(baseUrl string, uid string) string {
```

Comment:
to be consistent, maybe `ReadUserByUid`

Reply:
Hey this is used for both `read` and `update`, which is why I didn't call it `ReadUserByUid`.

Reply:
make sense, maybe just leave a comment above for that

Reply:
Renamed to `ReadOrUpdateUserByUid` to make it clearer and more consistent.